src/open_inwoner/scss/components/Questionnaire/Questionnaire.scss

```diff
@@ -34,7 +34,7 @@
     }
 
     &__list-item:nth-child(2) {
-      border-top: var(--border-width) solid var(--color-mute);
+      border-top: var(--border-width-thin) solid var(--color-gray);
```

Comment:
This changes only the second child each time. Let's set the border to your new variable for all the children.

![image](https://user-images.githubusercontent.com/63058071/218733710-b92e418d-60f0-4a14-9a1e-677a56a9fdb6.png)

Reply:
Oh wow, I was not very smart with this - i will correct it.